src/main/resources/feedbackQuestionContribAdditionalInfoTemplate.html

```diff
@@ -1,5 +1,14 @@
-&nbsp;<span style=" white-space: normal;">
-  <a href="javascript:;" id="questionAdditionalInfoButton-${questionNumber}-${additionalInfoId}" class="small link-in-dark-bg" onclick="toggleAdditionalQuestionInfo('${questionNumber}-${additionalInfoId}')" data-more="${more}" data-less="${less}">${more}</a>
-  <br>
-  <span id="questionAdditionalInfo-${questionNumber}-${additionalInfoId}" style="display:none;">${questionAdditionalInfo}</span>
-</span>
+&nbsp; <a href="javascript:;" id="more-info-contrib-calculation-modal-link" data-toggle="modal" data-target="#more-info-contrib-calculation-modal"><span class="glyphicon glyphicon-info-sign glyphicon-flipped"></span></a>
+<div id="more-info-contrib-calculation-modal" class="modal fade" role="dialog">
+  <div class="modal-dialog">
```

Comment:
There's quite a bit of content inside this modal, so the modal seems quite long vertically. Possible to make the modal wider?